sql/cagg_utils.sql

```diff
@@ -51,8 +51,8 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.drop_dist_ht_invalidation_trigg
 --                      Access Node that belong to 'raw_hypertable_id'
 -- bucket_widths - The array of time bucket widths for all the CAGGs that belong to
 --                 'raw_hypertable_id'
--- max_bucket_widths - The array of the maximum time bucket widths for all the CAGGs that belong
---                     to 'raw_hypertable_id'
+-- max_bucket_widths - (Deprecated) This argument is ignored and is present only
```

Comment:
I am curious if we should remove this argument as well, it is a bit unclear how this argument would be used for backward compatibility otherwise

Reply:
It will be removed eventually, the entire signature of the function will be quite different. Please see "Variable-Sized Buckets and Timezones Support for Continuous Aggregates" RFC.

Reply:
We can't remove it, because the AN can be of an older version than the DN. The order of upgrades is, upgrade DN first, then AN last.